Tests/Common/Data/TradeBarConsolidatorTests.cs

```diff
@@ -130,11 +130,11 @@ public void OneMinuteAlwaysFiresEveryTimeOnMinueDataExceptFirstPoint()
 
             consolidator.Update(new TradeBar {Close = 2m, Time = reference.AddMinutes(1)});
             Assert.IsNotNull(consolidated);
-            Assert.AreEqual(2, consolidated.Close);
+            Assert.AreEqual(1, consolidated.Close);
 
             consolidator.Update(new TradeBar {Close = 3m, Time = reference.AddMinutes(2)});
             Assert.IsNotNull(consolidated);
-            Assert.AreEqual(3, consolidated.Close);
+            Assert.AreEqual(2, consolidated.Close);
```

Comment:
OK, so with this we're saying that the bar's range is closed on the left and open on the right:
for example - [10:00 - 10:01)
So data at 10:01 is not included in the bar starting at 10:00.

I think this makes sense. Can we please update the comment in this same function to reflect the new behavior, and we'll also want to change the name of this test to match the behavior as well.

There are plans to add a Period property to TradeBar so we can distinguish how much time each bar represents, once that happens I think this case becomes trivial.